Cannot compile under Windows

```
[...]
Z:\workspace\marytts\marytts-common\src\main\java\marytts\util\MaryUtils.java:62
3: error: <identifier> expected
                        '\u00ec', 'i', // ├¼
                                      ^
Z:\workspace\marytts\marytts-common\src\main\java\marytts\util\MaryUtils.java:62
4: error: <identifier> expected
                        '\u00ed', 'i', // ├¡
                                 ^
100 errors
:marytts-common:compileJava FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':marytts-common:compileJava'.
> Compilation failed; see the compiler error output for details.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug
option to get more log output.

BUILD FAILED
```

--- a/buildLogic.gradle
+++ b/buildLogic.gradle
@@ -21,6 +21,10 @@ configurations {
     }
 }
 
+tasks.withType(JavaCompile) {
+    options.encoding = 'UTF-8'
+}
+
 task integrationTest(type: Test) {
     group 'Verification'
     description 'Runs the integration tests.'

